extensions/standard-processors/processors/ExtractText.cpp

```diff
@@ -126,22 +125,22 @@ int64_t ExtractText::ReadCallback::process(const std::shared_ptr<io::BaseStream>
   if (sizeLimitStr.empty())
     size_limit = DEFAULT_SIZE_LIMIT;
   else if (sizeLimitStr != "0")
-    size_limit = std::stoi(sizeLimitStr);
+    size_limit = gsl::narrow_cast<size_t>(std::stoi(sizeLimitStr));
```

Comment:
It comes from the user and negative numbers shouldn't crash the program, but they are not "valid" either.

Reply:
I think throwing an exception in `onSchedule()` would be better than treating a limit of "-2" as a limit of 18.4 exabytes, but this is how the code was working before this PR, and I agree that changing it would be out of scope.

Reply:
Actually it would make sense to cast them to `size_t` and treat `-1` as "essentially no limit" instead of throwing. If you think this is a good idea, I'll change it to `static_cast<size_t>(...)`.

Reply:
changed to `static_cast` in 0c23d4e